include/grpcpp/ext/otel_plugin.h

```diff
@@ -26,24 +26,31 @@
 
 #include "absl/functional/any_invocable.h"
 #include "absl/status/status.h"
+#include "absl/status/statusor.h"
 #include "absl/strings/string_view.h"
 #include "opentelemetry/metrics/meter_provider.h"
 
 #include <grpc/support/metrics.h>
 #include <grpc/support/port_platform.h>
+#include <grpcpp/support/channel_arguments.h>
 
 namespace grpc {
 
 namespace internal {
 class OpenTelemetryPluginBuilderImpl;
-class OpenTelemetryPlugin;
 }  // namespace internal
 
 class OpenTelemetryPluginOption {
  public:
   virtual ~OpenTelemetryPluginOption() = default;
 };
 
+class OpenTelemetryPlugin {
```

Comment:
I'm thinking - mark this as an experimental API first. After we confirm that our users are happy with this, we can stabilize it.

Reply:
done

Reply:
done

Reply:
If this is experimental, we should probably put it in an `experimental` namespace.

Reply:
done